pyproject.toml

```diff
@@ -46,7 +46,7 @@ pyright = "^1.1.298"
 pytest-cov = "^4.0.0"
 isort = "^5.11.4"
 pytest-rerunfailures = "^11.1"
-starknet-devnet = "0.5.0a2"
+starknet-devnet = {git = "https://github.com/Shard-Labs/starknet-devnet", rev = "75188cc"}
```

Comment:
starknet-devnet is officially released on PyPI, so it can be used directly, not from git repo 🔥